.github/workflows/pytest-run.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
Should this not be changed "ci-test" so that the CI triggers for PRs/pushes to that branch? (I can quickly do it after merging, just want to check before breaking something)